shared-data/labware/schemas/3.json

```
@@ -648,6 +647,33 @@
             "z": {
               "description": "Always zero by convention. Z-coordinate of the footprint plane.",
               "type": "number"
+            },
+            "springDirectionalForce": {
+              "description": "Applicable when the labware contains a spring that influences its position when placed on a parent deck item.",
+              "type": "string"
+            }
+          }
+        },
+        "slotFootprintAsParent": {
+          "description": "The 2D plane on which a child deck item resides.",
+          "type": "object",
+          "required": ["backLeft", "frontRight", "z"],
+          "properties": {
+            "backLeft": {
+              "description": "(-x, +y), the back left corner of a labware's mating plane.",
+              "$ref": "#/definitions/vector2D"
+            },
+            "frontRight": {
+              "description": "(+x, -y), the back left corner of a labware's module plane.",
+              "$ref": "#/definitions/vector2D"
+            },
+            "z": {
+              "description": "The z-offset from the bottom of the labware to the plane on which a child deck item resides.",
```

Nit:

```suggestion
              "description": "The z-offset from this labware's origin to the plane on which a child deck item resides.",
```

If you imagine a hypothetical definition where the labware is floating 15mm above its origin for some reason, so `extents.backLeftBottom.z` is 15, we're expecting this `z` to be relative to the origin (like everything else), not relative to `extents.backLeftBottom.z`.